tictactoe.html

```diff
@@ -0,0 +1,53 @@
+<!--#
+    set var="title"     value="Tic Tac Toe"             --><!--#
+    set var="header"    value="Tic Tac Toe"             --><!--#
+    set var="subheader" value="Have fun!"               --><!--#
+    set var="tictactoe" value="active"                  --><!--#
+    set var="link"      value="tictactoe/tictactoe.css" --><!--#
+
+    include virtual="ssi/header.html" --><!--#
+    include virtual="ssi/nav.html"    -->
+
+<article>
+
+    <p id="next">
+        Next move:
+    </p>
+
+    <p id="player">
+        X
+    </p>
+
+    <div id="main">
+        <table id="game">
+            <tbody>
+                <tr>
+                    <td class="cell" id="0"></td>
```

Comment:
All these `class="cell"` that exist within the `<tbody>` block are redundant. You're repeating yourself. DRY. Remove all the `class="cell"` and instead use a CSS selector like: `article#ttt table td`